src/udev/udev-worker.c

```diff
@@ -51,12 +51,11 @@ int udev_get_whole_disk(sd_device *dev, sd_device **ret_device, const char **ret
                 return log_device_debug_errno(dev, r, "Failed to get sysname: %m");
 
         /* Exclude the following devices:
-         * For "dm-", see the comment added by e918a1b5a94f270186dca59156354acd2a596494.
          * For "md", see the commit message of 2e5b17d01347d3c3118be2b8ad63d20415dbb1f0,
          * but not sure the assumption is still valid even when partitions are created on the md
          * devices, surprisingly which seems to be possible, see PR #22973.
          * For "drbd", see the commit message of fee854ee8ccde0cd28e0f925dea18cce35f3993d. */
-        if (STARTSWITH_SET(val, "dm-", "md", "drbd"))
+        if (STARTSWITH_SET(val, "md", "drbd"))
```

Comment:
did anything change regarding dm's issues with us taking the lock? at the very least this requires some ok by dm people that we should relax that now.

Reply:
I share the same concern. While I don't anticipate issues with acquiring the lock, I'm not an expert on device mapper. It would be very helpful if someone with more familiarity in that area could assist.